Gaijinizer patch/Map104.txt

```diff
@@ -90,9 +93,10 @@ and pleasure that I had never felt before.#MTLed
 あれから数週間経って、その間にも数回繰り返しているが
 身体に異常はない、むしろ元気なくらいだ。
 > CONTEXT: Map104/events/7/pages/0/34/Dialogue
-...... Oh, okay! It has been several weeks since then, and
-has been repeated several times in the meantime
-There is no abnormality in the body, but rather it is fine.#MTLed
+...Oh but it's okay! It's been several weeks since then,
+and I've done it several more times in that period, but
+there's abnormalities in my body, if anything I feel even
+better.#Ron
```

Comment:
Minor flow:
>_**Oh... but it's okay! It's been several weeks since then,
and I've done it several more times in that period, but
there's no abnormalities in my body.
If anything I feel even better.#Ron**_